README.md

```diff
@@ -166,6 +167,62 @@ mysql::db { 'mydb':
 }
 ```
 
+### Install Percona server on Centos
+
+This example shows how to do a minimal installation of a Percona server on a Centos (has been tested on Puppet 4.4 / Centos 7 / Percona Server 5.7) including the Percona server, client and bindings.
+
+The configuration change here is really minimal.
+
+**Note:** The installation of the yum repository is not part of this package and is here only to show a full example of how you can install this.
+
+```puppet
+yumrepo { 'percona':
+  descr    => 'CentOS $releasever - Percona',
+  baseurl  => 'http://repo.percona.com/centos/$releasever/os/$basearch/',
+  gpgkey   => 'http://www.percona.com/downloads/percona-release/RPM-GPG-KEY-percona',
+  enabled  => 1,
+  gpgcheck => 1,
+}
+
+class {'mysql::server':
+  package_name     => 'Percona-Server-server-57',
+  service_name     => 'mysqld',
+  config_file      => '/etc/my.cnf',
+  includedir       => '/etc/my.cnf.d',
+  override_options => {
+    mysqld => {
+      log-error => '/var/log/mysqld.log',
+      pid-file  => '/var/run/mysqld/mysqld.pid',
+    },
+    mysqld_safe => {
+      log-error => '/var/log/mysqld.log',
+    },
+  }
+}
+
+# Note: this is not needed  in our case as installing Percona-Server-server-57
+# also installs Percona-Server-client-57. It is only here to show how to
+# install the Percona MySQL client
+class {'mysql::client':
+  package_name => 'Percona-Server-client-57',
+}
+
+class { 'mysql::bindings':
+  client_dev_package_name => 'Percona-Server-devel-57',
+  daemon_dev_package_name => 'Percona-Server-shared-57',
+}
```

Comment:
The daemon_dev are generally mysql-devel or mariadb-devel which are the development headers and static libraries.
Those would correspond (big picture) to: Percona-Server-devel-57
https://www.percona.com/doc/percona-server/5.7/installation/yum_repo.html

The client_dev would correspond in this case to mysql-libs (or mysql-community-libs or mysql-commercial-libs ) or mariadb-shared which are shared/dynamic client libraries.
Percona-Server-shared-57 package would correspond to that.
So keeping that logic I should invert those. :)